dart/lib/src/hub.dart

```diff
@@ -176,6 +178,13 @@ class Hub {
     return sentryId;
   }
 
+  Future<void> captureUserFeedback(UserFeedback userFeedback) async {
+    final item = _peek();
```

Comment:
I know its a draft but just to not forget, requires checks like `captureEvent` does, if SDK is enabled, etc

Reply:
And additionally it probably needs a check that the `userFeedback.sentryId` is not equal to `SentryId.empty()`.

Reply:
still pending

Reply:
There's a check for a disabled SDK and the UserFeedback ctor disallows the use of an empty ID. Are there more conditions which I need to consider?

Reply:
lets add a check anyway because at runtime we don't know if its empty or not, so we don't even call the API if its an empty eventId